src/serializers/blocks.ts

```diff
@@ -0,0 +1,66 @@
+// See: <https://html.spec.whatwg.org/#the-css-user-agent-style-sheet-and-presentational-hints>
+// From: https://www.npmjs.com/package/rehype-minify-whitespace
+const blocks = [
```

Comment:
I really appreciate the attention to detail moving this to an appropriate location following it's usage in `slateToDom` as well as `slatetoHtml`.